lightbeam/lightbeam.py

```diff
@@ -71,6 +72,7 @@ def __init__(self, config_file, logger=None, selector="*", exclude="", keep_keys
         self.resend_status_codes=resend_status_codes
         self.endpoints = []
         self.results = []
+        self.tester = Tester(self)
```

Comment:
I think the `Lightbeam` class no longer uses a `Tester`?

Reply:
Ugh, I had to add this back because actually `test_lightbeam.py` initiates the tests with `Lightbeam.tester.test()` so it is needed (for now, at least).

Reply:
Ahh I see. Well it's not a huge deal.

That said, I notice that the `Tester` class only has a `logger` and a `lightbeam` as members, and the `test` function as its only method. Is it possible that `Tester` isn't even needed?

Like, what if there was just a function…
```python
# lightbeam/test.py
def test(tests_dir, lightbeam, logger):
    ...
```
And then the text execution file would just have:
```python
lb = Lightbeam(...)
assert test(tests_dir, lb, lb.logger)
```

Obviously not a deal-breaker by any means but this would achieve a full decoupling of application code and test code, which I think is ideal.

Reply:
Good suggestion, implemented.

I put the `test()` function in `lightbeam/tests/suite.py` which seems better than the root, and I renamed it to `run_test_suite()` so that `pytest` wouldn't try to run the function directly. (`pytest` tests functions called `test()`, `test_*()`, and `*_test()`.)

`test_lightbeam.py` just sets up and invokes the test suite.